README.md

```diff
@@ -1 +1,216 @@
+<!-- - [Live Demo Link](https://ayomoses1.github.io/Shachah/) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+```sh
+ npm install webpack webpack-cli webpack-dev-server
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd your-folder
+  git clone https://github.com/AyoMoses1/todolist.git
+```
+
+### Install
+
+Install this project with:
+
+
+
+```sh
+  cd my-project
+  node version 16
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+- GitHub Pages
+- Netlify
+- Heroku
+- Cyclic
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@AyoMoses1](https://github.com/AyoMoses1)
+- Twitter: [@Ayo_Moses1](https://twitter.com/Ayo_Moses1)
+- LinkedIn: [Ayo Moses](https://www.linkedin.com/in/ayo-moses-493946184/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Responsive Application**
+- **Local storage Integration**
+- **Github Deployment**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please support by contributing to the codebase
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Cindy Shin for the awesome and inspiring design idea of this project. The design guidelines for this project was gotten from one of her most profound designs for a conference. Please find here a link to the <a href="https://creativecommons.org/licenses/by-nc/4.0/legalcode">license</a>, <a href="https://creativecommons.org/licenses/by-nc/4.0/#">disclaimer</a> and a link to the original <a href="https://www.behance.net/gallery/29845175/CC-Global-Summit-2015">design</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
```

Comment:
- Hey, Please, kindly remove all the placeholder text in the readme as these are instructions meant for you to follow in order to add content to your readme and they're not meant to be in the readme after you've updated the content on your readme. please, see them below to remove them.


![Screenshot from 2023-01-26 15-50-11](https://user-images.githubusercontent.com/59397159/214867435-891b241e-34fa-4611-ad8c-86c558c6ca94.png)
![Screenshot from 2023-01-26 15-50-00](https://user-images.githubusercontent.com/59397159/214867440-6023dda9-0cef-42ee-b9ea-7780f84d3721.png)
![Screenshot from 2023-01-26 15-49-41](https://user-images.githubusercontent.com/59397159/214867441-064df8cd-13e7-4ad6-8715-7586d848d9f0.png)